kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/SessionsResource.scala

```diff
@@ -322,6 +322,70 @@ private[v1] class SessionsResource extends ApiRequestContext {
     }
   }
 
+  @ApiResponse(
+    responseCode = "200",
+    content = Array(new Content(
+      mediaType = MediaType.APPLICATION_JSON)),
+    description = "Close an operation")
+  @DELETE
+  @Path("{sessionHandle}/operations/{operationHandle}")
+  def closeOperation(
+      @PathParam("sessionHandle") sessionHandleStr: String,
+      @PathParam("operationHandle") operationHandleStr: String): OperationHandle = {
+    val sessionHandle = getSessionHandle(sessionHandleStr)
+    val operationHandle = getOperationHandle(operationHandleStr)
+    try {
+      backendService.sessionManager.getSession(sessionHandle).closeOperation(operationHandle)
+      operationHandle
+    } catch {
+      case NonFatal(_) =>
+        throw new NotFoundException(s"Error closing an operation")
+    }
+  }
+
+  @ApiResponse(
+    responseCode = "200",
+    content = Array(new Content(
+      mediaType = MediaType.APPLICATION_JSON)),
+    description =
+      "Get an operation handle with a given session identifier and operation identifier")
+  @GET
```

Comment:
I don't get the meaning of this API, why shall we use an operation handle to get itself?

Reply:
Is the `operation status` here duplicated with the api below?
![image](https://user-images.githubusercontent.com/18065113/144558460-3c1af612-d7ab-4311-8964-cc4631ccc93a.png)

Reply:
Yeah, Maybe we can remove this API that you put here. This API is for supporting a pure Hive RPC service interface.